ajax/libs/mui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework clostest to the native app",
```

Comment:
I think everybody has his/her own presentation, so it'll be better to point out the "problem" or tell us what's the better point in yours.